plugins/engine-configuration/impl/pom.xml

```diff
@@ -77,6 +77,12 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpcore</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.hitachivantara.security.web</groupId>
+      <artifactId>csrf-token-service-client-java-httpclient-v4</artifactId>
+      <version>${csrf.token.service.client.version}</version>
```

Comment:
Version must be inherited from the parent pom, so please remove the `version` element.

Reply:
addressed the comments

Reply:
You should remove the version element altogether.
> so please remove the version element.

Reply:
Removed the version as requested